src/elements/chip-group/chip-group/chip-group.ts

```diff
@@ -0,0 +1,406 @@
+import { customElement, property } from 'lit/decorators.js';
+
+import { inputAutocompleteEvent } from '../../autocomplete.js';
+import { getNextElementIndex, isArrowKeyPressed } from '../../core/a11y.js';
+import { forceType, hostAttributes } from '../../core/decorators.js';
+import { EventEmitter } from '../../core/eventing.js';
+import {
+  type FormRestoreReason,
+  type FormRestoreState,
+  SbbDisabledMixin,
+  SbbFormAssociatedMixin,
+  SbbNegativeMixin,
+} from '../../core/mixins.js';
+import { SbbChipElement } from '../chip.js';
+
+import style from './chip-group.scss?lit&inline';
+
+export interface SbbChipInputTokenEndEventDetails {
+  /** The element that triggered the chip creation */
+  origin: 'input' | 'autocomplete';
+  /** The value of the new chip. Either the input or the option value depending on the origin */
+  value: string;
+  label?: string;
+  /** Set a new value for the chip that will be created */
+  setValue(value: string): void;
+  /** Set a label for the chip that will be created */
+  setLabel(value: string): void;
+}
+
+/**
+ * The `sbb-chip-group` component is used as a container for one or multiple `sbb-chip`.
+ *
+ * @event {CustomEvent<void>} change - Notifies that the component's value has changed.
+ * @event {CustomEvent<void>} input - Notifies that the component's value has changed.
+ * @event {CustomEvent<SbbChipInputTokenEndEventDetails>} chipInputTokenEnd - Notifies that a chip is about to be created. Can be prevented.
+ * @slot - Use the unnamed slot to add `sbb-chip` elements.
+ * @overrideType value - string[] | null
+ */
+export
+@customElement('sbb-chip-group')
+@hostAttributes({
+  tabindex: '0',
+  role: 'grid',
+})
+class SbbChipGroupElement extends SbbDisabledMixin(
+  SbbNegativeMixin(SbbFormAssociatedMixin<typeof LitElement, string[]>(LitElement)),
+) {
+  public static override styles: CSSResultGroup = style;
+  public static readonly events: Record<string, string> = {
+    input: 'input',
+    change: 'change',
+    chipInputTokenEnd: 'chipInputTokenEnd',
+  } as const;
+
+  /** Value of the form element. */
+  @property({ type: Array })
+  public override set value(value: string[] | null) {
+    value = value ?? [];
+    super.value = value;
+    const oldValue = this.value;
+
+    // Subtract from 'oldValue' the new 'value'
+    // The result are the chips to remove (handle duplicates)
+    const toRemove = [...oldValue];
+    for (const c of value) {
+      if (toRemove.includes(c)) {
+        toRemove.splice(toRemove.indexOf(c), 1);
+      }
+    }
+    toRemove.forEach((value) =>
+      this._chipElements()
+        .find((c) => c.value === value)
+        ?.remove(),
+    );
+
+    // Subtract from the new 'value' what was already present
+    // The result are the new chips to add (handle duplicates)
+    const toAdd = [...value];
+    for (const c of oldValue) {
+      if (toAdd.includes(c)) {
+        toAdd.splice(toAdd.indexOf(c), 1);
+      }
+    }
```

Comment:
Could this be replaced by:
```suggestion
    const toAdd = newValue.filter(item => !oldValue.includes(item));
```

Reply:
Same as previous feedback